token_auth/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
+
+from token_auth.views.register import RegisterView
+
+
+urlpatterns = [
+    path('register/', RegisterView.as_view()),
+    # path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    # path('refresh/', TokenRefreshView.as_view(), name='token_refresh'),
```

Comment:
Was this for testing out something? If so, shouldn't it be deleted?

Reply:
No just API enpoints which will be in use later!